app/models/Cocktail.php

```diff
@@ -3,7 +3,7 @@ class Cocktail
 {
     private $cocktail_id;
     private $user_id;
-    public $hasLiked = false;
+    public $hasLiked = null;
```

Comment:
I noticed $hasLiked is set to null now instead of false. Is there a specific reason for this? Usually, it’s either true or false to track whether the cocktail is liked, so having null might add some unnecessary complexity unless we need it to mean something like ‘unknown’ or ‘not fetched yet. What's the reason for this change ?

Reply:
Upon refreshing a page, even the cocktails that had been liked by the user, would display with a white heart. Only after reapplying like, the heart color would change to red.
My deduction lead to this change. Of course if your system does not reflect the same issue as mine, I would need to reinvestigate why this happens.